docs/components/input-controller/_index.md

```diff
@@ -34,7 +34,6 @@ Most machines with an input controller need at least the following hardware:
 
 {{< cards >}}
 {{< relatedcard link="/mobility/base-rc/" >}}
-{{< relatedcard link="/mobility/frame-system/" >}}
```

Comment:
Can you think of any other related services that might be good to replace this with?

Reply:
My thought is no, for the same reason that I don't think frame system should be here--I'm thinking of navigation/motion/frame system as related to each other and useful for automating motion, whereas I think of input control as almost the opposite? Can you think of any?

Reply:
I don't think any others make sense so I'm going to merge!